app/src/main/java/com/hillnerds/soundsparrow/SoundGeneration.java

```diff
@@ -10,135 +18,88 @@
 
 public class SoundGeneration {
```

Comment:
Why is this class still a thing? From what I can see, all methods are static. This should be combined with ChannelSoundGeneration I think